jck/build.xml

```diff
@@ -87,156 +88,16 @@
-									<then>
-										<echo message="Deleting ${JCK_ROOT_USED}/natives..." />
-										<delete includeEmptyDirs="true">
-  											<fileset dir="${JCK_ROOT_USED}/natives"/>
-										</delete>
-									</then>
-								</if>
-								<echo message="Updating ${JCK_ROOT_USED} with latest..." />
-								<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false" resultproperty="return.code">
-									<arg value="pull" />
-									<arg value="${JCK_GIT_REPO_USED}" />
-									<arg value="${jck_branch}" />
-								</exec>
-								<if>
-									<not>
-										<equals arg1="${return.code}" arg2="0"/>
-									</not> 
-									<then>
-										<echo message="Performing hard reset..." />
-										<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
-											<arg value="reset" />
-											<arg value="--hard" />
-											<arg value="origin/main" />
-										</exec>
-										<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
-											<arg value="clean" />
-											<arg value="-f" />
-											<arg value="-d" />
-										</exec>
-										<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
-											<arg value="pull" />
-											<arg value="${JCK_GIT_REPO_USED}" />
-											<arg value="${jck_branch}" />
-										</exec>
-									</then>
-								</if>
-							</else>
-						</if>
-					</else>
-				</if>
-			</else>
-		</if>
-		<if>
-			<isset property="isZOS" />
-			<then>
-				<if>
-					<isset property="local-material-uptodate"/>
-					<then> 
-						<echo message="Local JCK materials up-to-date. Skipping hard reset"/>
-					</then>
-					<else>
-						<echo message="Performing hard reset of ${JCK_ROOT_USED} using .gitattributes.zos for file conversions..."/>
-						<delete includeemptydirs="true" failonerror="false">
-							<fileset dir="${JCK_ROOT_USED}/.git/info" includes="**/*"/>
-						</delete>
-						<mkdir dir="${JCK_ROOT_USED}/.git/info" />
-						<move file="${JCK_ROOT_USED}/.gitattributes.zos" tofile="${JCK_ROOT_USED}/.git/info/attributes" />
-						<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false">
-							<arg value="rm" />
-							<arg value="--cached" />
-							<arg value="-r" />
-							<arg value="-q" />
-							<arg value="." />
-						</exec>
-						<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false">
-							<arg value="reset" />
-							<arg value="--hard" />
-						</exec>
-					</else>
-				</if>
-			</then>
-		</if> 
+		<exec executable="bash" failonerror="false">
+			<arg value="update_git_material.sh"/>
+			<!-- Add other arguments as needed -->
+			<!-- Pass all properties as environment variables -->
+			<!-- Pass all properties as environment variables -->
+			<env key="JCK_ROOT_USED" value="${JCK_ROOT_USED}" />
+			<env key="JCK_GIT_REPO_USED" value="${JCK_GIT_REPO_USED}" />
+			<env key="jck_branch" value="${jck_branch}" />
+			<env key="isZOS" value="${isZOS}" />
```

Comment:
Setting `env` variables are not necessary here, you can pass in them as parameters of the shell script, you can refer this test for it ([link](https://github.com/adoptium/aqa-tests/blob/c089b7328234947ed6cc7d24632c084049da5e5c/external/criu-portable-checkpoint/build.xml#L21-L27))